garden_ai/utils/filesystem.py

```diff
@@ -31,6 +40,17 @@ def load_pipeline_from_python_file(python_file: Path) -> Pipeline:
     module = importlib.util.module_from_spec(spec)
     try:
         spec.loader.exec_module(module)
+    except ScaffoldedModelException as e:
+        raise ScaffoldedModelException(
+            "Failed to load model. It looks like you are trying to register a scaffolded pipeline."
+            + " Please replace YOUR MODEL's NAME HERE in your pipeline.py with the name of a registered Garden model."
+            + "\nFor more information on how to use Garden, please read our docs: https://garden-ai.readthedocs.io/en/latest/"
+        ) from e
+    except MlflowException as e:
```

Comment:
I think MLflow exceptions are broader than this. I think it's possible we could see an MLflowException because of, say, an auth error or a transient network error. I think you _could_ know that it's not the name of a registered model if it has a 404 error code.